src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java

```diff
@@ -392,4 +417,12 @@ public static void waitForLoading() {
       );
     }
   }
+
+  public static void tryToCloseAlert() {
+    try {
+      ElementHelper.clickOnElement(Helper.locateElementByTestId("alert-close"));
+    } catch (NoSuchElementException e) {
```

Comment:
Why is it needed to catch exception and not throwing any error?

Reply:
From my development experience, it seems sometimes there is an alert banner but sometimes not. So I have this helper function to close the banner if there is one.